### DISCUSSION TITLE
RSS/ Atom news feed input

### DISCUSSION THREAD

**Discussion by dantefromhell** [omnivore-app/omnivore#1958] [Q&A]
I am curious if RSS/ Atom based subscriptions are a planned feature for Omnivore?

I love the email subscription feature for newsletters a lot, but many of the technology blogs I post unregularly w/o email lists - but they all offer RSS/ Atom based news feeds.

**Comment by tomdavidson** [comment]
This feature exists right?

![image](https://github.com/omnivore-app/omnivore/assets/50412/64565933-38fa-45ef-b798-5b7c2a4c3231)

**Reply by camflan** [reply]
Yep, that works but there's 0 support in the app.

2 enhancements that would be good,
1. When adding a link, detect if its an RSS feed
2. When adding a URL to the page you've captured above, if the URL provided is not a feed, it should load the link and look for a feed URL to use
